src/Atmos/Parameterizations/CloudPhysics/Activation.jl

```diff
@@ -0,0 +1,154 @@
+
+
+Pkg.add("SpecialFunctions")
```

Comment:
```suggestion
using SpecialFunctions
```

Reply:
This should also be moved inside the module it's used, rather than coming before it (since this file is `include`d somewhere else). And same elsewhere :slightly_smiling_face:.